deploy/values.yml

```diff
@@ -0,0 +1,92 @@
+replicaCount: ${REPLICA_COUNT}
+service:
+  enabled: true
+  name: ${APP_NAME}
+
+  port: ${PORT}
+  type: ClusterIP
+  containerPort: ${PORT}
+  tls:
+    enabled: true
+    issuerName: cloudflare
+
+image:
+  repository: ${ECR_REGISTRY}/${ECR_REPO_NAME}
+  tag: ${IMAGE_TAG}
+  pullPolicy: Always
+
+serviceAccount:
+  create: false
+
+ingress:
+  enabled: true
+  hosts:
+    - ${ENDPOINT}
+  annotations:
+    external-dns.alpha.kubernetes.io/cloudflare-proxied: "true"
+
+httpBasicAuth:
+  enabled: false
+
+# If enabled will create Traefik Middleware and apply to Ingress
+# to redirect http to https and www to non-www
+httpWwwRedirect:
+  enabled: false
+
+podAnnotations:
+  reloader.stakater.com/auto: "true"
+
+resources:
+  limits:
+    cpu: 1.2
+    memory: 1.5Gi
+  requests:
+    cpu: 750m
```

Comment:
Keep it consistent when specifying CPU resources - decide here on milli or cores.